[2017-10-14 12:53:16] php.DEBUG: Warning: fsockopen(): unable to connect to 127.0.0.1:8000 (Connection refused) {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/var/www/html/flyaround/vendor/symfony/symfony/src/Symfony/Bundle/WebServerBundle/WebServerConfig.php\",\"line\":120,\"trace\":[{\"file\":\"/var/www/html/flyaround/vendor/symfony/symfony/src/Symfony/Bundle/WebServerBundle/WebServerConfig.php\",\"line\":54,\"function\":\"findBestPort\",\"class\":\"Symfony\\\\Bundle\\\\WebServerBundle\\\\WebServerConfig\",\"type\":\"->\"}],\"count\":1})"} []
[2017-10-14 12:53:22] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2017-10-14 12:53:22] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-10-14 12:53:22] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_controller":"web_profiler.controller.profiler:toolbarAction","token":"9f1a66","_route":"_wdt"},"request_uri":"http://localhost:8000/_wdt/9f1a66","method":"GET"} []
[2017-10-14 12:53:31] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2017-10-14 12:53:31] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-10-14 12:53:31] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_controller":"web_profiler.controller.profiler:toolbarAction","token":"596532","_route":"_wdt"},"request_uri":"http://localhost:8000/_wdt/596532","method":"GET"} []
[2017-10-14 13:14:20] console.ERROR: Error thrown while running command "doctrine:database:create". Message: "An exception occured in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES)" {"error":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES) at /var/www/html/flyaround/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:103, Doctrine\\DBAL\\Driver\\PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES) at /var/www/html/flyaround/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES) at /var/www/html/flyaround/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)","command":"doctrine:database:create","message":"An exception occured in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES)"} []
[2017-10-14 13:14:20] console.DEBUG: Command "doctrine:database:create" exited with code "1" {"command":"doctrine:database:create","code":1} []
[2017-10-14 13:16:14] console.ERROR: Error thrown while running command "doctrine:database:create". Message: "An exception occured in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES)" {"error":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES) at /var/www/html/flyaround/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:103, Doctrine\\DBAL\\Driver\\PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES) at /var/www/html/flyaround/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES) at /var/www/html/flyaround/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)","command":"doctrine:database:create","message":"An exception occured in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES)"} []
[2017-10-14 13:16:14] console.DEBUG: Command "doctrine:database:create" exited with code "1" {"command":"doctrine:database:create","code":1} []
[2017-10-14 13:16:48] console.ERROR: Error thrown while running command "doctrine:database:create". Message: "An exception occured in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES)" {"error":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES) at /var/www/html/flyaround/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:103, Doctrine\\DBAL\\Driver\\PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES) at /var/www/html/flyaround/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES) at /var/www/html/flyaround/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)","command":"doctrine:database:create","message":"An exception occured in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES)"} []
[2017-10-14 13:16:48] console.DEBUG: Command "doctrine:database:create" exited with code "1" {"command":"doctrine:database:create","code":1} []
[2017-10-14 13:21:03] php.DEBUG: Warning: fsockopen(): unable to connect to 127.0.0.1:8000 (Connection refused) {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/var/www/html/flyaround/vendor/symfony/symfony/src/Symfony/Bundle/WebServerBundle/WebServerConfig.php\",\"line\":120,\"trace\":[{\"file\":\"/var/www/html/flyaround/vendor/symfony/symfony/src/Symfony/Bundle/WebServerBundle/WebServerConfig.php\",\"line\":54,\"function\":\"findBestPort\",\"class\":\"Symfony\\\\Bundle\\\\WebServerBundle\\\\WebServerConfig\",\"type\":\"->\"}],\"count\":1})"} []
[2017-10-14 13:21:33] console.ERROR: Error thrown while running command "doctrine:database:create". Message: "An exception occured in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES)" {"error":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES) at /var/www/html/flyaround/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:103, Doctrine\\DBAL\\Driver\\PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES) at /var/www/html/flyaround/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES) at /var/www/html/flyaround/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)","command":"doctrine:database:create","message":"An exception occured in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES)"} []
[2017-10-14 13:21:33] console.DEBUG: Command "doctrine:database:create" exited with code "1" {"command":"doctrine:database:create","code":1} []
[2017-10-14 13:33:54] console.ERROR: Error thrown while running command "doctrine:database:create". Message: "An exception occured in driver: SQLSTATE[HY000] [1045] Access denied for user 'phpmyadmin'@'localhost' (using password: YES)" {"error":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[HY000] [1045] Access denied for user 'phpmyadmin'@'localhost' (using password: YES) at /var/www/html/flyaround/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:103, Doctrine\\DBAL\\Driver\\PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'phpmyadmin'@'localhost' (using password: YES) at /var/www/html/flyaround/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'phpmyadmin'@'localhost' (using password: YES) at /var/www/html/flyaround/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)","command":"doctrine:database:create","message":"An exception occured in driver: SQLSTATE[HY000] [1045] Access denied for user 'phpmyadmin'@'localhost' (using password: YES)"} []
[2017-10-14 13:33:54] console.DEBUG: Command "doctrine:database:create" exited with code "1" {"command":"doctrine:database:create","code":1} []
[2017-10-14 13:43:12] console.DEBUG: Command "doctrine:database:create" exited with code "1" {"command":"doctrine:database:create","code":1} []
